### DISCUSSION TITLE
what is the meaning of “block.size”？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#151] [Q&A]
what is the meaning of “block.size”？
iris.obj <- rfsrc(Species ~ ., data = iris, block.size = 1, importance = TRUE)

**Comment by ishwaran** [comment]
This is clearly defined in the help file for `rfsrc.`



`**block.size**  Should the cumulative error rate be calculated on every tree? When NULL, it will only be calculated on the last tree and the plot of the cumulative error rate will result in a flat line. To view the cumulative error rate on every nth tree, set the value to an integer between 1 and ntree. As an intended side effect, if importance is requested, VIMP is calculated in "blocks" of size equal to block.size, thus resulting in a useful compromise between ensemble and tree VIMP. The default action is to use 10 trees.
`
